_posts/2021-03-23-keccak-optimizer-bug.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+published: true
+title:  "Solidity Keccak Caching Bug"
+date:   2021-03-22
```

Comment:
I'll update the date tomorrow. Otherwise, the post won't show up in netlify deploy.